### ISSUE TITLE
custom folder and the possibility of a new connection without closing the GUI

### ISSUE THREAD

**Issue by TOWUK** [mircokroon/minecraft-world-downloader#396]
hello, there are some difficulties, for example, I entered the address and connected to the server, he downloaded the chunks and saved them to a folder next to the application, is it possible to somehow specify the folder of worlds immediately in the minecraft game? for example `%APPDATA%/.minecraft/saves`
and also sometimes they don’t pass checks on proxy servers and writes that I can’t log in through VPN or proxy
to enter a new server, I need to close the application, although you can go to the settings and change the ip, but this is not allowed, the fields are not editable when already connected to the server

**Comment by mircokroon** [comment]
> is it possible to somehow specify the folder of worlds immediately in the minecraft game?

Yes, you can just put the full path in the world location field. E.g. `C:\Users\USERNAME\AppData\Roaming\.minecraft\saves\world`


> although you can go to the settings and change the ip, but this is not allowed, the fields are not editable when already connected to the server

I'm not sure I understand the use case for changing the server IP after you've already started downloading the world

**Comment by TOWUK** [comment]
I would like to try the system of worlds like this: for example, multi worlds are used on the server, world-downloader creates a folder with the name of the server's IP and it already downloads each world separately, as on the server itself, because it happens that I go into the lobby, download it and then I go into minigames and the application rewrites the world, which is not very convenient for me to load them in local worlds later
